coq/goals.mli

```diff
@@ -40,6 +40,24 @@ type 'a goals =
   ; given_up : 'a list
   }
 
+type pp_tag = [%import: Pp.pp_tag] [@@deriving yojson]
+type block_type = [%import: Pp.block_type] [@@deriving yojson]
+
+module Pretty : sig
+  type t =
+    | Pp_empty
+    | Pp_string of string
+    | Pp_glue of t list
+    | Pp_box of block_type * t
+    | Pp_tag of pp_tag * t
+    | Pp_print_break of int * int
+    | Pp_force_newline
+    | Pp_comment of string list
+  [@@deriving yojson]
```

Comment:
SerAPI already provides the yojson for `Pp.t` , indeed the seralization should happen in the lsp layer not here

Reply:
I thought `coq-lsp` would subsume `serapi` ... I read it somewhere in the README.

Reply:
Yes, it will subsume serapi, not serlib tho, sorry for the confusion.

Reply:
Also note that for now we have kept the `coq` library free of deps, so it could be merged in upstream at some point.

Reply:
Fixed.